src/__tests__/videoList.test.js

```diff
@@ -0,0 +1,28 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+
+import VideoList from '../components/VideoList';
+import VideoContext from '../context/videoContext';
+import items from '../mocks/youtube-videos-mock.json'
+
+describe('VideoList component tests', () => {
+
+  test('should containt a list container', () => {
+    render(<VideoContext><VideoList videos={items.items}/></VideoContext>);
+    const list = screen.getByTestId(/container_videos/i);
+    expect(list).toBeInTheDocument();
+  });
+
+  test('should containt a card video list', () => {
+    render(<VideoContext><VideoList videos={items.items}/></VideoContext>);
+    const list = screen.findAllByTestId(/list_videos/i);
+    expect(list.length).toBe(25);
```

Comment:
As I understand, you could put 

```js
expect(list.length).toBe(items.length);
```

in order to have more resilient to changes this test (for example, if someone else modifies the youtube-videos-mock.json content to 10 videos only)